core/server/models/post.js

```diff
@@ -129,12 +186,12 @@ Post = ghostBookshelf.Model.extend({
 
         // ### Business logic for published_at and published_by
         // If the current status is 'published' and published_at is not set, set it to now
-        if (this.get('status') === 'published' && !this.get('published_at')) {
-            this.set('published_at', new Date());
+        if (newStatus === 'published' && !publishedAt) {
+            this.set('published_at', moment.utc().toDate());
```

Comment:
It seems to me that this change may need some wider thought?

If we're changing one date in the database to be definitely UTC, rather than server time, then we should probably do this for all dates? Including `created_at` and `updated_at` timestamps throughout the data model, and we'd need to do some form of migration to convert all the existing dates?

Possibly a very, very rare case that server time is not set to UTC though, meaning that's probably a lot of work for an edgecase? :confused: